apps/user/src/app/(main)/map/components/MapContainer.tsx

```
@@ -19,27 +18,25 @@ export default function MapContainer() {
-    handleFetchCategories();
-    setSelectedCategory({ ...ALL_CATEGORY });
-  }, []);
+    getCategories().then(({ data, error }) => {
+      if (error) {
+        console.log("카테고리 로딩 실패: ", error);
+        // 에러 상태 관리 또는 토스트 알림 추가
+        return;
+      }
+      if (data && data.data && Array.isArray(data.data.data.categoryList)) {
+        setCategories([
+          ALL_CATEGORY,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)